roles/ansible/tower/config-ansible-tower-subscription-manifest/README.md

```diff
@@ -0,0 +1,53 @@
+config-ansible-tower-subscription-manifest
+=========================
+
+This role is used to provide an Ansible Tower instance with a subscription manifest
+
+## Requirements
+
+A running Ansible Tower with admin permission level access.
+
+
+## Role Variables
+
+The variables used to configure Ansible Tower LDAP are outlined in the table below.
+
+| Variable | Description | Required | Defaults |
+|:---------|:------------|:---------|:---------|
+|ansible_tower.admin_password|Admin password for the Ansible Tower install|yes||
+|ansible_tower.admin_username|Admin username for the Ansible Tower install|yes||
```

Comment:
this should be "Required: no" and the default value is `admin`

Reply:
Updated. Please note I have also added the retries/delay as options in the README.